feature/curation/src/main/java/com/linku/curation/CurationViewModel.kt

```
@@ -1,221 +1,14 @@
-
-            }catch (e: Exception){
-                Log.d("CurationViewModel", "clearRecentQuery catch: $e.message")
-            }finally {
-                Log.d("CurationViewModel", "clearRecentQuery finally")
-            }
-        }
-        Log.d("CurationViewModel", "clearRecentQuery return")
-    }
-    // ---------- search method ----------
+) : ViewModel() {
 }
```

detekt (1.23.8)

[warning] The class or object CurationViewModel is empty.

(detekt.empty-blocks.EmptyClassBlock)